src/redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+const CHECKED_STATUS = 'CHECKED_STATUS';
```

Comment:
- [ ] Also modify the CHECKED_STATUS action type to match the ducks pattern in order to complete this project's requirement.
 - **Hint**: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`